src/lightning_app/core/flow.py

```diff
@@ -608,3 +603,36 @@ def experimental_iterate(self, iterable: Iterable, run_once: bool = True, user_k
             yield value
 
         self._calls[call_hash].update({"has_finished": True})
+
+    def configure_commands(self):
+        """Configure the commands of this LightningFlow.
+
+        Returns a list of dictionaries mapping a command name to a flow method.
+
+        .. code-block:: python
+
+            class Flow(LightningFlow):
+                ...
+                def __init__(self):
+                    super().__init__()
+                    self.names = []
+
+                def handle_name_request(name: str)
+                    self.names.append(name)
+
+                def configure_commands(self):
+                    return [
+                        {"add_name": self.handle_name_request}
+                    ]
+
+        Once the app is running with the following command:
+
+        .. code-block:: bash
+
+            lightning run app app.py command
+
+        .. code-block:: bash
+
+            lightning run command add_name --args name=my_own_name
```

Comment:
Which app will this run the command for? Are you taking the name from the .lightning file?
I need to be able to set the name via --name. Does that work?